components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2982,6 +2982,14 @@ impl Animatable for IntermediateSVGPaint {
         Ok(self.kind.compute_squared_distance(&other.kind)? +
             self.fallback.compute_squared_distance(&other.fallback)?)
     }
+
+    #[inline]
+    fn get_zero_value(&self) -> Option<Self> {
+        Some(IntermediateSVGPaint {
+            kind: option_try!(self.kind.get_zero_value()),
+            fallback: self.fallback.map(|v| v.get_zero_value().unwrap()),
```

Comment:
Thanks @nox ,

If we remove this unwrap(), this map will return Option<Option<IntermediateRGBA>> value since fallback field type is Option<IntermediateRGBA>, So I added unwrap().

For detail, please see https://reviewboard.mozilla.org/r/150652/#review156568.